packages/engine/README.md

```diff
@@ -28,8 +28,9 @@
   * [CLI Arguments and Options](#cli-arguments-and-options)
   * [Run a simulation](#run-a-simulation)
   * [Simulation Outputs](#simulation-outputs)
-    + [JSON-State](#json-state)
-    + [Analysis](#analysis)
+    + [JSON-State](#json-state-json_statejson)
```

Comment:
What's going on here and on line 32?

Reply:
The headings are called <code>JSON-State [`json_state.json`]</code>, thus the extra "-json_statejson` is required.

Reply:
Yeah just funky internal link specifics with how it strips characters and such

Reply:
Although that said, Tim are you sure the underscore stays?

Reply:
Weird, discussion on Github about this implies it should be stripped. But checked the branch and it works:
https://github.com/hashintel/hash/tree/td/behavior-readme/packages/engine#analysis-analysis_outputsjson

Good enough for me